### PR TITLE
Make filter box a selection box, read selections aloud

### LINKED ISSUE #92: Filter box does not announce selections
The only indication that one has selected something from the filters is that the checkbox is marked. However, for people with low or no vision, this indication will not be useful - the accessibility label will have to change to announce the selections.

### PR DESCRIPTION
**Pull request by carpenterd777**
Changed the filter box to be a selection box as it was discovered to be possibly more intuitive to have a system where clicking the checkbox opts the user in rather than the other way around. Also, the show/hide filter button will tell the user what they currently have selected so that they do not need to rely solely on visual indication.

Closes #92

### CODE REVIEW

File: src/components/PlaceList/PlaceList.tsx [hunk 1]
```diff
@@ -1,18 +1,12 @@
...
-const deepEqual = (a: Place[], b: Place[]): boolean => {
-	return JSON.stringify(a) === JSON.stringify(b);
-};
-
 const PlaceList = (): JSX.Element => {
-	//const [places, setPlaces] = useState<Place[]>([]);
 	const { nearbyPlaces } = useNearbyPlaces();
 
-	const cardList = (nearbyPlaces || []).map((value, index) => {
+	const cardList = (nearbyPlaces?.slice(0, 5) || []).map((value, index) => {
```
Comment (aej11a) [inline src/components/PlaceList/PlaceList.tsx:9]: why this change? I don't think we need to limit the number of cards to 5, we only do that in nearby places for readability and relevance while strolling

**Review by aej11a — CHANGES_REQUESTED** [review]

File: src/components/PlaceList/PlaceList.tsx (hunk 1 above)
Comment (carpenterd777) [reply src/components/PlaceList/PlaceList.tsx:9]: It seemed like a lot of places at once to me, but I'll change it back since it really doesn't hurt much

Comment (aej11a) [reply src/components/PlaceList/PlaceList.tsx:9]: @carpenterd777 still waiting on this one

**Review by aej11a — APPROVED** [review]
